docs/sources/guides/whats-new-in-v6-0.md

```diff
@@ -130,7 +130,7 @@ The Azure Monitor datasource integrates four Azure services with Grafana - Azure
 
 Grafana now added support for provisioning alert notifiers from configuration files. Allowing operators to provision notifiers without using the UI or the API. A new field called `uid` has been introduced which is a string identifier that the administrator can set themselves. Same kind of identifier used for dashboards since v5.0. This feature makes it possible to use the same notifier configuration in multiple environments and refer to notifiers in dashboard json by a string identifier instead of the numeric id which depends on insert order and how many notifiers that exists in the instance.
 
-## Easier to deploy & more secure
+## Easier to deploy & more secure authentication
```

Comment:
Why? My thought is that we've improved almost all our cookie settings, not only related to authentication. Doesn't really matter to me though, but if you change this you'll need to change the hash of the link at the top.

Reply:
`more secure` felt vague. how about `Easier to deploy & improved security`?